### PR TITLE
Handle miscellaneous warnings

### PR DESCRIPTION
**Pull request by stevensoftware52**
Not including plugins

### CODE REVIEW

File: UI/auth-oauth.cpp [hunk 1]
```diff
@@ -64,6 +64,8 @@ OAuthLogin::OAuthLogin(QWidget *parent, const std::string &url, bool token)
 	QVBoxLayout *topLayout = new QVBoxLayout(this);
 	topLayout->addWidget(cefWidget);
 	topLayout->addLayout(bottomLayout);
+#else
+	(void)url; // Avoid compiler warining about unused variable on windows
```
Comment (summeroff) [inline UI/auth-oauth.cpp]: obs has a macros UNUSED_PARAMETER(url); 
if it available in UI code better to use it here too.

File: plugins/text-freetype2/text-freetype2.c [hunk 2]
```diff
@@ -454,7 +454,7 @@ static void ft2_source_update(void *data, obs_data_t *settings)
 	if (srcdata->font_face)
 		cache_standard_glyphs(srcdata);
 
-skip_font_load:
+//skip_font_load:
```
Comment (summeroff) [inline plugins/text-freetype2/text-freetype2.c]: comment probably left here after experiment with warning.

**Review by summeroff — APPROVED** [review]

File: UI/auth-oauth.cpp (hunk 1 above)
Comment (stevensoftware52) [reply UI/auth-oauth.cpp]: Thanks, used this.

File: plugins/text-freetype2/text-freetype2.c (hunk 2 above)
Comment (stevensoftware52) [reply plugins/text-freetype2/text-freetype2.c]: Thanks, corrected

**Review by EddyGharbi — APPROVED** [review]

**Pull request merged**